### DISCUSSION TITLE
AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue.

### DISCUSSION THREAD

**Discussion by ucartz** [AzuraCast/AzuraCast#5670] [General]
when we try to add custom port through "docker-compose" and docker composer restarted , we are getting below error .

azuracast  | [2022-08-13T05:14:32.840889+00:00] AzuraCast.WARNING: Console command `azuracast:acme:get-certificate` exited with error code 1. [] []
azuracast  | [2022-08-13T05:14:32.851447+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":1,"name":"8000"}}
azuracast  | [13-Aug-2022 05:14:32] NOTICE: fpm is running, pid 100
azuracast  | [13-Aug-2022 05:14:32] NOTICE: ready to handle connections
azuracast  | [13-Aug-2022 05:14:32] NOTICE: systemd monitor interval set to 10000ms
azuracast  | 2022-08-13 05:14:33,408 INFO success: php-nowplaying entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
azuracast  | [2022-08-13T05:14:33.566179+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":11,"name":"8020"}}
azuracast  | [2022-08-13T05:14:34.234923+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":12,"name":"9090"}}
azuracast  | [2022-08-13T05:14:34.936581+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":16,"name":"8050"}}
azuracast  | [2022-08-13T05:14:35.675779+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":18,"name":"8002"}}
azuracast  | [2022-08-13T05:14:36.378653+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":19,"name":"8052"}}
azuracast  | [2022-08-13T05:14:37.061580+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":20,"name":"8004"}}
azuracast  | [2022-08-13T05:14:37.722309+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":21,"name":"8054"}}
azuracast  | [2022-08-13T05:14:39.421248+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":1,"name":"8000"}}
azuracast  | [2022-08-13T05:14:41.857922+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":11,"name":"8020"}}
azuracast  | [2022-08-13T05:14:42.513248+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":19,"name":"8052"}}
azuracast  | [2022-08-13T05:14:45.199911+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":12,"name":"9090"}}
azuracast  | [2022-08-13T05:14:45.873607+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":18,"name":"8002"}}
azuracast  | [2022-08-13T05:14:47.566823+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":16,"name":"8050"}}
azuracast  | [2022-08-13T05:14:48.972119+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":20,"name":"8004"}}
azuracast  | [2022-08-13T05:14:49.653630+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":21,"name":"8054"}}
azuracast  | [2022-08-13T05:14:51.319959+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":1,"name":"8000"}}
azuracast  | [2022-08-13T05:14:52.702275+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":11,"name":"8020"}}
azuracast  | [2022-08-13T05:14:53.395758+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":12,"name":"9090"}}
azuracast  | [2022-08-13T05:14:54.113599+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":19,"name":"8052"}}
azuracast  | [2022-08-13T05:14:55.789321+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":18,"name":"8002"}}
azuracast  | [2022-08-13T05:14:57.459623+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":16,"name":"8050"}}
azuracast  | [2022-08-13T05:15:00.154249+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":11,"name":"8020"}}
azuracast  | [2022-08-13T05:15:00.973273+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":21,"name":"8054"}}
azuracast  | [2022-08-13T05:15:02.616467+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":1,"name":"8000"}}
azuracast  | [2022-08-13T05:15:03.252420+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":12,"name":"9090"}}
azuracast  | [2022-08-13T05:15:03.966060+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":18,"name":"8002"}}
azuracast  | [2022-08-13T05:15:04.675289+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":19,"name":"8052"}}
azuracast  | [2022-08-13T05:15:05.341379+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":20,"name":"8004"}}
azuracast  | [2022-08-13T05:15:07.025990+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":16,"name":"8050"}}
azuracast  | [2022-08-13T05:15:09.734962+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":11,"name":"8020"}}
azuracast  | [2022-08-13T05:15:10.445562+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":12,"name":"9090"}}
azuracast  | [2022-08-13T05:15:11.180203+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":21,"name":"8054"}}
azuracast  | [2022-08-13T05:15:12.928626+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":1,"name":"8000"}}
azuracast  | [2022-08-13T05:15:16.277902+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":19,"name":"8052"}}
azuracast  | [2022-08-13T05:15:17.969535+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":11,"name":"8020"}}
azuracast  | [2022-08-13T05:15:18.721630+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":16,"name":"8050"}}
azuracast  | [2022-08-13T05:15:19.405654+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":18,"name":"8002"}}
azuracast  | [2022-08-13T05:15:20.116987+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":20,"name":"8004"}}
azuracast  | [2022-08-13T05:15:20.809588+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":21,"name":"8054"}}
azuracast  | [2022-08-13T05:15:23.526455+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":12,"name":"9090"}}
azuracast  | [2022-08-13T05:15:24.209752+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":19,"name":"8052"}}
azuracast  | [2022-08-13T05:15:25.872073+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":1,"name":"8000"}}
azuracast  | [2022-08-13T05:15:27.266536+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":11,"name":"8020"}}
azuracast  | [2022-08-13T05:15:27.974029+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":16,"name":"8050"}}
azuracast  | [2022-08-13T05:15:29.713861+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":20,"name":"8004"}}
azuracast  | [2022-08-13T05:15:30.417571+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":21,"name":"8054"}}
azuracast  | [2022-08-13T05:15:33.103750+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":18,"name":"8002"}}
azuracast  | [2022-08-13T05:15:33.827220+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":19,"name":"8052"}}
azuracast  | [2022-08-13T05:15:35.550378+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":11,"name":"8020"}}
azuracast  | [2022-08-13T05:15:36.231266+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":12,"name":"9090"}}
azuracast  | [2022-08-13T05:15:38.898930+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":16,"name":"8050"}}
azuracast  | [2022-08-13T05:15:39.621129+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":20,"name":"8004"}}
azuracast  | [2022-08-13T05:15:40.355133+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":21,"name":"8054"}}
azuracast  | [2022-08-13T05:15:42.044648+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":1,"name":"8000"}}
azuracast  | [2022-08-13T05:15:44.746509+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":19,"name":"8052"}}
azuracast  | [2022-08-13T05:15:46.441171+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":11,"name":"8020"}}
azuracast  | [2022-08-13T05:15:48.109389+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":12,"name":"9090"}}
azuracast  | [2022-08-13T05:15:48.813140+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":18,"name":"8002"}}
azuracast  | [2022-08-13T05:15:49.506084+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":20,"name":"8004"}}
azuracast  | [2022-08-13T05:15:51.209588+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":16,"name":"8050"}}
azuracast  | [2022-08-13T05:15:52.068557+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":21,"name":"8054"}}
azuracast  | [2022-08-13T05:15:53.751662+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":19,"name":"8052"}}
azuracast  | [2022-08-13T05:15:55.447193+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":11,"name":"8020"}}
azuracast  | [2022-08-13T05:15:57.126332+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":1,"name":"8000"}}
azuracast  | [2022-08-13T05:15:58.869741+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":12,"name":"9090"}}
azuracast  | [2022-08-13T05:15:59.544785+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":20,"name":"8004"}}
azuracast  | [2022-08-13T05:16:01.237177+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":18,"name":"8002"}}
azuracast  | [2022-08-13T05:16:01.913652+00:00] AzuraCast.NOTICE:

**Comment by kemecs** [comment]
I have the same experience:

[2022-08-28T08:03:23.977657+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":1,"name":"stream.folkradio.hu"}}
[2022-08-28T08:03:38.096825+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":1,"name":"stream.folkradio.hu"}}
.....

Rolling Release #[53080b1] (2022-08-23 4:36) • Docker • PHP 8.1


There may be a connection:
https://github.com/AzuraCast/AzuraCast/issues/5698
